src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/cas2/Cas2ApplicationsTransformerTest.kt

```diff
@@ -123,7 +131,7 @@ class Cas2ApplicationsTransformerTest {
       )
 
       assertThat(result.id).isEqualTo(application.id)
-      assertThat(result.createdBy.id).isEqualTo(user.id)
+      assertThat(result.createdBy.id).isEqualTo(this@Cas2ApplicationsTransformerTest.nomisUserEntity.id)
```

Comment:
A kotlin language thing, shouldn't be there:

```suggestion
      assertThat(result.createdBy.id).isEqualTo(nomisUserEntity.id)
```